native/src/modules/app/createReduxStore.js

```diff
@@ -60,7 +62,11 @@ const createReduxStore = (
     resourceCacheUrl: resourceCacheUrlReducer
   })
 
-  const middleware = applyMiddleware(sagaMiddleware)
+  const middlewares = [sagaMiddleware]
+  if (__DEV__) {
```

Comment:
Use the build config here: `const middleware = buildConfig().development ? flipperReduxMiddleware`